learn/what_is_meilisearch/telemetry.md

```diff
@@ -4,33 +4,41 @@ sidebarDepth: 2
 
-**Amplitude** collects the following datapoints:
+- **System**: technical specs of the device and OS running MeiliSearch, MeiliSearch version, etc.
+- **Performance**: success rate of search requests, latency, etc.
+- **Usage**: adoption of new features, usage rate of features we're considering changing, etc.
 
-- Amount of time the MeiliSearch instance has been running
-- MeiliSearch version
-- Technical specs of the device running MeiliSearch
-- Database size per instance
-- Number of documents per instance
+**We will never:**
 
-**We will absolutely never identify users, track users, or collect private information**, including IP address, email, or website URLs. All of our data collection is **GDPR-compliant** and **secure**.
+- Identify or track users
+- Collect private information such as IP addresses, email addresses, or website URLs
+- Store data from the content of documents added to MeiliSearch
 
-::: tip
-In the future, we may collect additional metrics relating to new features. This page will be kept up-to-date, but for the most precise information be sure to check the release notes after each new version. <Badge text="soon" type="warn"/>
-:::
+See below for an [exhaustive list of metrics we collect](#exhaustive-list-of-all-collected-data).
 
 ## Why collect telemetry data?
 
-We collect anonymized data for only two reasons: so that we can improve our product, and so that we can continue working on this project full-time.
+We collect telemetry data for only two reasons: so that we can improve our product, and so that we can continue working on this project full-time.
+
+In order to create a better product, we need reliable quantitative information. The information we collect helps us fix bugs, evaluate the success of features, and better understand our users' needs.
 
-In order to better address our users' needs and chart a clear path forward for MeiliSearch, we need reliable quantitative information. The information we collect helps us fix bugs, evaluate the success of certain features, and identify our strengths and weaknesses as a product.
+We also need to prove that people are actually using MeiliSearch. Usage metrics help us justify our existence to investors so that we can keep this project alive.
 
-We also need to prove that people are actually using MeiliSearch! Usage metrics help us justify our existence to investors so that we can keep this project alive.
+## Why should you trust us?
+
+The short answer: you shouldn't. We feel that it is perfectly understandable, and in fact wise, to be distrustful of tech companies when it comes to your private data. That is why we attempt to maintain [complete transparency about our data collection](#exhaustive-list-of-all-collected-data), provide an [opt-out](#how-to-disable-data-collection), and allow [deletion of collected data](#how-to-delete-all-collected-data) at any time. We believe that this is the only way.
```

Comment:
Thanks @gmourier , I like it better than what I wrote!

Come join the technical writing team please, we have cookies 😁 🍪

Reply:
Ended up not using this suggestion, since it felt a little contradictory to start the paragraph saying "don't trust us", and end it saying that we're doing this to build trust.